crates/uv-resolver/src/resolver/batch_prefetch.rs

```diff
@@ -3,7 +3,7 @@ use std::cmp::min;
 use itertools::Itertools;
 use pubgrub::range::Range;
 use rustc_hash::FxHashMap;
-use tokio::sync::mpsc::Sender;
+use tokio::sync::mpsc::UnboundedSender;
```

Comment:
Why the switch to an unbounded channel? I'm not usually a fan of these because it prevents back-pressure. Maybe use `crossbeam-channel` instead?

Reply:
I can switch back to bounded channels, tokio has a `blocking_send` method. Now that things are running in parallel I don't see a huge benefit of the solver blocking on the prefetcher unless it's necessary, but I agree that having some backpressure is a good idea.